aurt/tests/api_tests.py

```diff
@@ -65,15 +58,6 @@ def test03_compile_rd(self):
 
         api.compile_rd(model_rbd, friction_torque_model, friction_viscous_powers, output_path, self.cache)
 
-        with open(output_path, 'rb') as f:
-            rd_twolink_estimate = pickle.load(f)
-        # with open(from_project_root(Path("aurt/tests/resources", output_file)), 'rb') as f:
-        #     rd_twolink_true = pickle.load(f)
-
-        # self.assertEqual(rd_twolink_estimate.n_joints, rd_twolink_true.n_joints)
-        # self.assertEqual(rd_twolink_estimate.qdd, rd_twolink_true.qdd)
-        # self.assertEqual(rd_twolink_estimate.tauJ, rd_twolink_true.tauJ)
```

Comment:
nothing is being asserted in this test. Is it possible to compare to values to check if the generated rd model is correct?

Reply:
Those three lines in particular does not make sense to keep. But yes, it would be possible to compare something, I'm just not sure what exactly.

From my memory, we actually removed tests that compared symbolic expressions because they were breaking easily - like for instance if variables were renamed. Although, that could be circumvented by evaluating the symbolic robot dynamics (e.g. the observation matrix) numerically using e.g. some arbitrary measurements, for instance:
- np.abs(rd_twolink_estimate.observation_matrix(arbitrary_measurements) - rd_twolink_old.observation_matrix(arbitrary_measurements)) <= 1e-8